ethereum/vm.py

```diff
@@ -135,6 +131,25 @@ def data_copy(compustate, size):
     return True
 
 
+def eat_gas(compustate, amount):
+    if compustate.gas < amount:
+        compustate.gas = 0
+        return False
+    else:
+        compustate.gas -= amount
+        return True
+
+
+def all_but_1_nth(gas, n):
+    return gas - (gas // n)
+
+
+def max_call_gas(gas):
+    """Since EIP150 CALLs will send only all but 1/64th of the available gas.
+    """
+    return all_but_1_nth(gas, n=opcodes.CALL_CHILD_LIMIT_DENOM)
```

Comment:
Having the increased readability of the `all_but_1_nth` function might IMO not be worth the additional function call overhead.